build.gradle

```diff
@@ -255,6 +281,62 @@ project('segmentstore:server:host') {
     }
 }
 
+
+project('segmentstore:server:hostwithbindings') {
```

Comment:
I don't see how `hostwithbindings` gets included in the distribution or the docker images. It is including the one without the bindings now.

I think you need to ship them regardless as part of the bundle. Some projects like flink have a `contrib`, `opt` or `extras` directory with extra non-required jar files. There still needs to be an easy process to get these onto the classpath.